client/blocks/user-mentions/suggestion-list.jsx

```diff
@@ -6,13 +6,13 @@ import PopoverMenuItem from 'calypso/components/popover-menu/item';
 import './suggestion-list.scss';
 
 const UserMentionsSuggestionList = ( {
-	onClick,
-	onClose,
-	popoverContext,
-	popoverPosition,
-	query,
-	selectedSuggestionId,
-	suggestions,
+	onClick = () => {},
+	onClose = () => {},
```

Comment:
No need for the default here, `Popover` will already provide its own default (although it can also be avoided with optional chaining)